modules/schematics/src/effect/files/__path__/__name@dasherize@if-flat__/__name@dasherize__.effects.ts

```diff
@@ -6,7 +6,7 @@ import { Actions, Effect } from '@ngrx/effects';
 export class <%= classify(name) %>Effects {
 <% if(feature) { %>
   @Effect()
-  effect$ = this.actions$.ofType(<%= classify(name) %>ActionTypes.<%= classify(name) %>Action);
+  effect$ = this.actions$.ofType(<%= classify(name) %>ActionTypes.Load<%= classify(name) %>s);
```

Comment:
@brandonroberts - you may want to add this:

```ts
load<%= classify(name) %>sEffect$ = this.actions$.ofType(<%= classify(name) %>ActionTypes.Load<%= classify(name) %>s);
```

Reply:
Good idea.